host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent

```diff
@@ -0,0 +1,56 @@
+upstream glitchtip {
+   server localhost:50504;
+   keepalive 32;
+}
+
+server {
+    listen 80;
+    server_name glitchtip.pratchett.zeus.gent;
+
+    if ($request_method = GET) {
+        return 301 https://$server_name$request_uri;
+    }
+
+    return 308 https://$server_name$request_uri;
+}
+
+server {
```

Comment:
Ik had dit blok nog niet gezien, dat mag ook weg, gezien we geen gltichtip.zeus.gent gaan doen

Reply:
Gaan we dat niet doen?